### ISSUE TITLE
ERD for Northwind db

### ISSUE THREAD

**Issue by DickBaker** [markjprice/cs12dotnet8#71]
suggestion to replace existing Figure 10.1 for more visual/logical learners
![Northwind-major8](https://github.com/user-attachments/assets/0e0525e7-ff1a-40d4-b7a8-8d504e47e54d)

looking forward to the new .NET-9 release !

**Comment by markjprice** [comment]
I agree that it can be visually useful for the relationships lines to begin and end at the columns that are part of the foreign key relationship, although standard conceptual ER diagrams do not show this information. For the next editions of my books that use Northwind, I plan to use Mermaid for the ER diagram, as shown at the following link:
https://github.com/markjprice/cs13net9/blob/main/docs/ch10-er-diagram.md
As you can see, it cannot show which columns are defined in the relationship. I will consider manually using SQL Server database diagrams again to manually define the figure, and include a link to the online Mermaid diagram as an extra.

**Comment by DickBaker** [comment]
Mark, Hi

 

Seen the Mermaid diagram and agree that it is secular-pink rich but information-poor

- hope you err on the side of information rather than ideology :)

 

Best Regards,

Dick
